clean_ioc/core.py

```diff
@@ -1357,7 +1357,7 @@ def register(
         tags: Iterable[Tag] | None = None,
         parent_node_filter: NodeFilter = default_parent_node_filter,
         scoped_teardown: Callable[[TService], Any] | None = None,
-    ) -> None: ...
+    ) -> Scope: ...
```

Comment:
Cannot have a return type Scope here because a `Scope` is a `Registrator`.
This makes it's type graph cyclic.

Reply:
I have removed the return scope and rebased onto upsteam